website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/index.mdx

```diff
@@ -92,47 +97,35 @@ It requires the following ServiceNow plugins as dependencies:
-perform the following steps to configure and gather information from Terraform
-Cloud.
+perform the following steps to configure and gather information from HCP
+Terraform.
 
 1. Either [create an
    organization](/terraform/cloud-docs/users-teams-organizations/organizations#creating-organizations)
    or choose an existing organization where ServiceNow will create new
    workspaces.
    **Save the organization name for later.**
-1. [Create a team](/terraform/cloud-docs/users-teams-organizations/teams) for that
+2. [Create a team](/terraform/cloud-docs/users-teams-organizations/teams) for that
    organization called "ServiceNow", and ensure that it has [permission to
    manage
    workspaces](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-workspaces).
    You do not need to add any users to this team.
    [permissions-citation]: #intentionally-unused---keep-for-maintainers
-1. On the "ServiceNow" team's settings page, generate a [team API
+3. On the "ServiceNow" team's settings page, generate a [team API
    token](/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens).
    **Save the team API token for later.**
-1. If you haven't yet done so, [connect a VCS provider](/terraform/cloud-docs/vcs) to
-   this Terraform organization. This provider will include one or more
-   repositories with Terraform configurations. Any repository that could be
-   connected to a manually-created HCP Terraform workspace can also be
-   used as a workspace template in the ServiceNow integration.
-1. On your organization's VCS provider settings page (Settings > VCS Providers),
-   find the "OAuth Token ID" for the VCS provider or providers you will be using
-   with the ServiceNow integration. HCP Terraform uses the OAuth token ID to
-   identify and authorize the VCS provider.
-   **Save the OAuth token ID for later.**
-1. Identify one or more VCS repositories in the VCS provider containing
-   Terraform configurations that will be provisioned by the ServiceNow Terraform
-   integration. Take note of any Terraform or environment variables used by the
-   repositories you select.
-   **Save the Terraform and environment variables for later.**
-1. Provide the following information to the ServiceNow Admin.
-1. The organization name
-1. The team API token
-1. The hostname of your Terraform Enterprise instance, or of HCP Terraform.
-   The hostname of HCP Terraform is `app.terraform.io`.
-1. The OAuth token ID(s) of your VCS provider(s), and the repository
-   identifier for each VCS repository containing Terraform configurations that
-   will be used by the integration.
-1. Any Terraform or environment variables required by the configurations in the
+4. If you are using the [VCS flow](/terraform/cloud-docs/integrations/service-now/service-catalog-terraform/service-catalog-config#vcs-configuration): 
+    1. If you haven't yet done so, [connect a VCS
```

Comment:
I think you can omit the second sentence here because you link out to a place that contains more information. I tried shortening this a bit for brevity but please feel free to push back if anything doesn't feel good:
```suggestion
    1. Ensure your Terraform organization is [connected a VCS provider](/terraform/cloud-docs/vcs). Repositories that are connectable to HCP Terraform workspaces can also be used as workspace templates in the ServiceNow integration.
```

Reply:
(Leaving this as one comment rather than spacing it out over the lines because it's easier to read, but if you accept, this is a reminder to delete the lines below!)